yt/frontends/gamer/tests/test_outputs.py

```diff
@@ -97,3 +102,32 @@ def test_jiw():
     for test in small_patch_amr(ds, _fields_jiw):
         test_jiw.__name__ = test.description
         yield test
+
+
+@requires_ds(jiw, big_data=True)
+def test_stress_energy():
+    axes = "txyz"
+    ds = data_dir_load(jiw)
+    center = ds.arr([3.0, 10.0, 10.0], "kpc")
+    sp = ds.sphere(center, (1.0, "kpc"))
+    c2 = ds.units.clight**2
+    inv_c2 = 1.0 / c2
+    rho = sp["gas", "density"]
+    p = sp["gas", "pressure"]
+    e = sp["gas", "thermal_energy_density"]
+    gamma = sp["gas", "lorentz_factor"]
+    h = rho + (e + p) * inv_c2
+    T00 = gamma * gamma * c2 * h - p
+    assert_array_almost_equal(sp["T00"], T00)
+    for mu in range(4):
+        for nu in range(4):
+            # matrix is symmetric so only do the upper-right part
+            if nu > mu:
+                Umu = sp[f"four_velocity_{axes[mu]}"]
+                Unu = sp[f"four_velocity_{axes[nu]}"]
+                Tmunu = h * Umu * Unu
+                if mu != nu:
+                    assert_array_almost_equal(sp[f"T{mu}{nu}"], sp[f"T{nu}{mu}"])
+                else:
+                    Tmunu += p
```

Comment:
this is unreachable given `nu > mu` is checked on this branch.

Reply:
Thanks!